.travis/push_maps_yaml

```diff
@@ -9,16 +9,14 @@ git config --global user.name "tripleabuilderbot"
 ## clone the website repo, suppress any output
 git clone --quiet https://${PUSH_TO_WEBSITE_TOKEN}@github.com/triplea-game/triplea-game.github.io.git website 2>&1 > /dev/null
 
-## copy in the new maps.yaml
-cp ./triplea_maps.yaml ./website/_data/
+#Split the yaml file into multiple files
+./.travis/yaml_splitter ./triplea_maps.yaml ./website/_maps/
 
 ## do git stuff, check if there is a diff, if so, commit and push it
 cd website
-UPDATED=1; git status | grep "nothing to commit" && UPDATED=0
-
-if [ "$UPDATED" -eq 1 ]; then
+if ! git diff-index --quiet HEAD --; then
```

Comment:
I changed this line because I noticed that the old command is not working on non-english systems, because the message `nothing to commit` is in another language...